features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -191,11 +192,10 @@ export function SidebarNavigation() {
           </LinkList>
 
           <List>
-            <MenuItemButton
+            <ContactSupport
```

Comment:
You're one of the first who used a link in the first round 👍 Just one hint: The task is even simpler than this. You could use the existing `MenuItemLink` component as in the `<LinkList>` above. No need to create a new component

Reply:
I'll have to give that a shot tomorrow! Should have realized when I basically copied the component over there was a bit too much repetition 😁